alf/utils/process_coordinator_test.py

```diff
@@ -406,10 +400,11 @@ def decrement(self):
         m.share_memory()
         coord = coordinator.Coordinator()
         p = MyProcess(coord, kwargs={"m": m})
+        p.start()
+        time.sleep(0.01)  # sleep just enough for init to execute
```

Comment:
init is executed in the main process and it will always end before p.start(). I guess this sleep is not necessary and the comment is confusing.

Reply:
Good point.  I just want to test changing share_memory after subprocess started, but before subprocess start decrementing the value.  Changed comment.  PTAL.